spec/models/question_spec.rb

```diff
@@ -15,14 +15,16 @@
 
       it { expect(assignment.solution).to be nil }
       it { expect(assignment.messages.count).to eq 1 }
-      it { expect(assignment.submission_id).to_not be nil }
+      it { expect(assignment.submission_id).to be nil }
+      it { expect(assignment.submitted_at).to be nil }
     end
 
     context 'when a question on a previous submission is sent' do
       before do
-        assignment = exercise.submit_solution!(student, content: 'x = 1')
+        assignment = exercise.submit_solution!(student, content: 'x = 1').reload
```

Comment:
This `reload` call is tricky. It was necessary since otherwise the time was truncated in the database and results were not the same in line 37.